src/client/Client.cc

```diff
@@ -8115,18 +8115,14 @@ loff_t Client::lseek(int fd, loff_t offset, int whence, const UserPerm& perms)
   if (f->flags & O_PATH)
     return -EBADF;
 #endif
-  return _lseek(f, offset, whence, perms);
+  return _lseek(f, offset, whence);
 }
 
-loff_t Client::_lseek(Fh *f, loff_t offset, int whence, const UserPerm& perms)
+loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
 {
   Inode *in = f->inode.get();
   int r;
 
-  if (!may_open(f->inode.get(), f->flags, perms)) {
-    return -EPERM;
-  }
```

Comment:
No. I'm not sure how to tell from the teuthology logs -- could be anything really. This did deal in java after all, and I'm not 100% sure how the jre handles process cred changes. Might two threads have hit a name collision and be running different tests vs. the same files?

Also, the EPERM here clobbers the return from may_open, which might have told us a bit more about the failure. If you're curious we could change that to log a warning when that occurs without failing the operation, and see if we can catch it in the act?